src/kurtosis.rs

```diff
@@ -0,0 +1,195 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+use arrow::array::{ArrayRef, Float64Array, UInt64Array};
+use arrow::datatypes::{DataType, Field};
+use datafusion::arrow;
+use std::any::Any;
+use std::fmt::Debug;
+
+use datafusion::common::cast::as_float64_array;
+use datafusion::common::downcast_value;
+use datafusion::common::DataFusionError;
+use datafusion::error::Result;
+use datafusion::logical_expr::function::{AccumulatorArgs, StateFieldsArgs};
+use datafusion::logical_expr::{Accumulator, AggregateUDFImpl, Signature, Volatility};
+use datafusion::scalar::ScalarValue;
+
+make_udaf_expr_and_func!(
+    KurtosisFunction,
+    kurtosis,
+    x,
+    "Calculates the excess kurtosis (Fisher’s definition) with bias correction according to the sample size.",
+    kurtosis_udaf
+);
+
+pub struct KurtosisFunction {
+    signature: Signature,
+}
+
+impl Debug for KurtosisFunction {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        f.debug_struct("KurtosisFunction")
+            .field("signature", &self.signature)
+            .finish()
+    }
+}
```

Comment:
I think `#[derive(Debug)]` macro should serve the purpose here.

Reply:
It seems that an explicit implementation of Debug will be more informative.